conf/application.conf

```diff
@@ -103,13 +113,13 @@ microservice {
       clientId = "client_id_must_be_set_in_app-config-xxx"
       clientSecret = "client_secret_must_be_set_in_app-config-xxx"
 
-      # # Use for local testing with the real HIP API:
+      # Use for local testing with the real HIP API:
       # protocol = "https"
       # host = "admin.qa.tax.service.gov.uk"
       # port = 443
       # ref-data-path = "hip/crdl/views/iv_crdl_reference_data"
       # customs-offices-path = "hip/crdl/views/iv_crdl_customs_office"
-      # # The following environment variables must be set using credentials from Integration Hub:
+      # The following environment variables must be set using credentials from Integration Hub:
```

Comment:
If you leave these double-commmented things here, you can use comment / uncomment shortcuts on the whole block in editors without turning those comments into invalid lines